### ISSUE TITLE
训练模型复现

### ISSUE THREAD

**Issue by dshwei** [RUCKBReasoning/codes#20]
您好，我按照您提供的训练table schema 的训练脚本和 sql generate 的训练脚本训练数据集采用bird-evidence 的数据
执行命令如下：

**schema linker:使用的是A800  80G 单卡**
python -u train_schema_item_filter.py \
    --batch_size 4 \
    --gradient_descent_step 8 \
    --device 0 \
    --learning_rate 1e-5 \
    --gamma 2.0 \
    --alpha 0.75 \
    --epochs 64 \
    --patience 8 \
    --seed 42 \
    --save_path ./sic_ckpts/sic_bird_with_evidence \
    --tensorboard_save_path ./train_logs/sic_bird_with_evidence \
    --train_filepath ./sft_bird_with_evidence_train_text2sql.json \
    --dev_filepath ./sft_bird_with_evidence_dev_text2sql.json \
    --model_name_or_path roberta-large \
    --mode train
    
    
    
    模型的结果不能复现,模型结果不能复现，模型测试结果只有46.81，请问什么原因，训练过程的超参数有其他的修改吗
    
![企业微信截图_1718347818491](https://github.com/RUCKBReasoning/codes/assets/42167236/d8f94d0b-72d2-4e35-a7e1-984c9da23883)

**Comment by lihaoyang-ruc** [comment]
schema linker在dev集上的auc值是多少？以及sql generate用的多大的CodeS模型？
